README.md

```diff
@@ -55,9 +54,9 @@ brew install iter8
 You can also install Iter8 using:
 * [pre-compiled binaries](https://iter8.tools/latest/getting-started/install/)
 * [source](https://iter8.tools/latest/getting-started/install/)
-* [`go 1.16+`](https://iter8.tools/latest/getting-started/install/)
+* [`go 1.17+`](https://iter8.tools/latest/getting-started/install/)
 
-## Usage Examples
+## Usage
 
 ### HTTP load test with SLOs
```

Comment:
The links in the example have changed; should be updated.

Reply:
I verified that all links in this document are valid.